salt/states/lvm.py

```diff
@@ -265,7 +265,7 @@ def lv_present(
     force
         Assume yes to all prompts
 
-    .. versionadded:: to_complete
+    .. versionadded:: 3002.0
```

Comment:
```suggestion
    .. versionadded:: 3002.2
```

Reply:
The resizefs was added in 3002.0 but I forgot to update the versionadded, you can see it [here](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.lvm.html#salt.states.lvm.lv_present). I only saw that when updating the documentation description, so I fixed it in this commit to point to the correct version.

Reply:
Thanks for pointing that out.